Clarifying the behavior of PSA sync (#50310)
openshift/openshift-docs: modules/security-context-constraints-psa-synchronization.adoc

[IMPORTANT]
====
Namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. These namespaces include:

* `default`
* `kube-node-lease`
* `kube-system`
* `kube-public`
* `openshift`
* All system-created namespaces that are prefixed with `openshift-`, except for `openshift-operators`

> By default, all namespaces that have an `openshift-` prefix are not synchronized. You can enable synchronization for any user-created [x-]`openshift-*` namespaces by setting the value of the `security.openshift.io/scc.podSecurityLabelSync` label to `true`. You cannot enable synchronization for any system-created [x-]`openshift-*` namespaces, except for `openshift-operators`.

Reviewer: ```suggestion
By default, {product-title} will not synchronize namespaces that have an `openshift-` prefix. 
```
Does it make sense to reword this to avoid passive voice?

Reviewer 2: The original passive voice still sounds the best to me here
